src/index.css

```diff
@@ -1,3 +1,335 @@
 body {
   margin: 0;
+  font-family:
+    -apple-system,
+    BlinkMacSystemFont,
+    'Segoe UI',
+    'Roboto',
+    'Oxygen',
+    'Ubuntu',
+    'Cantarell',
+    'Fira Sans',
+    'Droid Sans',
+    'Helvetica Neue',
+    sans-serif;
+  -webkit-font-smoothing: antialiased;
+  -moz-osx-font-smoothing: grayscale;
+  padding: 4%;
+  background-color: #f5f6fa;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
- [ ] Great styling however you shouldn't leave the gray space in the corners and top and bottom please make sure to remove it so it can be like any normal web app 
![Screenshot 2023-01-25 at 17 21 50](https://user-images.githubusercontent.com/61790587/214619051-c940ffec-9b3b-4e5f-a63d-318f11069cbc.png)